tox.ini

```diff
@@ -1,8 +1,8 @@
-# Copyright (c) 2017-18 Richard Hull and contributors
+# Copyright (c) 2017-2019 Richard Hull and contributors
 # See LICENSE.rst for details.
 
 [tox]
-envlist = py{27,34,35,36,37},qa,doc
+envlist = py{27,34,35,36,37,38},qa,doc
```

Comment:
Remove 34 from envlist also?

Reply:
@rm-hull at first I thought this would be a good idea but it's still in stretch/jessie and having ability to run tests using tox is still useful, even when it's officially EOL.

Reply:
i removed it.